src/SSL.md

```diff
@@ -87,4 +87,23 @@ ClientConfig clientConfig = new ClientConfig();
 clientConfig.getNetworkConfig().addAddress("127.0.0.1");
 ```
 
-![image](images/NoteSmall.jpg) ***NOTE:*** *When you use SSL with the Java client, it will have a throughput that is 50% of a non-SSL Java client with the same configuration. If it is a .NET client, it will have a throughput that is 46% of a non-SSL .NET client.*
```

Comment:
I have removed this section since I don't know how valid these numbers are. Also with 3.8 a lot of things have changed + with the right changes there is some additional performance to gain on the JVM.